pybossa/otp.py

```diff
@@ -73,3 +73,10 @@ def retrieve_email_for_token(token):
 def expire_token(token):
     key = _create_url_token_key(token)
     conn.delete(key)
+
+def is_enabled(user_email, config):
+    # Allow bypassing two-factor-auth if the feature is disabled or if the user's email is included in the bypass list.
+    bypass_list = config.get('BYPASS_TWO_FACTOR_AUTH', [])
+    enable_two_factor_auth = config.get('ENABLE_TWO_FACTOR_AUTH', False)
+
+    return enable_two_factor_auth and (bypass_list is None or not user_email in bypass_list)
```

Comment:
This is good - fyi you can also do `... or user_email not in bypass_list` - also if you don't want to check if `bypass_list` is `None` you can do
`bypass_list = config.get('BYPASS_TWO_FACTOR_AUTH') or []`

Reply:
Thank you. That was the syntax I was looking for. :)
https://github.com/bloomberg/pybossa/pull/269/commits/a6892133c8398e36fe0bd4518ae8d1b5dccf2c6a